extension/llm/export/builder.py

```diff
@@ -394,9 +415,12 @@ def export_to_edge(self) -> "LLMEdgeManager":
                     return_value=False,
                 )
 
+            # Prior to export, persist the changes to the pre autograd
+            # graph module back to the source-of-truth ExportedProgram.
+            self.export(self.pre_autograd_graph_module)
```

Comment:
I think we should keep exported_program up-to-date. Thus shouldnt do this here but rather wherever we extract `graph_module` and apply any transformations. Thus we should not keep `self.pre_autograd_graph_module` at all. Only source of truth would be exported_program